integration-tests/field-validation/src/index.exo

```diff
@@ -8,4 +8,13 @@ module RuntimeValidations {
     @range(min = "-1.0", max = "3.0") float_limit: Float?
     price: Float?
   }
+
+  @access(true)
+  type OtherNumbers {
+    @pk id: Int = autoIncrement()
+    @range(min = 0, max = 12345) limit: Int?
+    age: Int?
+    @range(min = "-1.0", max = "12345.0") float_limit: Float?
```

Comment:
May be change `min` to a different value, just in case.

Also, it will be good to add more tests for updates for both positive and negative cases.